### DISCUSSION TITLE
Longcows are different on different planets

### DISCUSSION THREAD

**Discussion by tuuktuuk** [endless-sky/endless-sky#10780] [Lore]
I was on some planet where I learned that longcows grow a new segment every season and are mostly docile. Now I'm on Miblulub's plenty, and I'm told that longcows are "multi-legged" and "ill-behaved".

Why did the other planet mention nothing about the multiple legs?! And don't they concentrate on segment growth in Miblubub? Why not? Why are Miblubub's longcows worse-tempered than others? Or are the people in Miblubub's plenty so docile that even docile animals seem ill-behaved from their perspective?

**Comment by Arachi-Lover** [comment]
Planet descriptions aren't meant to give global information about everything they mention, rather one planet's description complements the other.

Growing segments each season and having various legs aren't mutually exclusive, but rather, well, complement each other. More segments = more legs.
Longcows are animals, so there being a planet where their herds are more docile and another where they're more ill-behaved seems pretty normal  - IRL cows can vary a good deal in how docile they are, based on their breeds. No reason to think longcows wouldn't have breeds of their own with such differences.

**Reply by tuuktuuk** [reply]
I somewhat agree. But in that case, throwing in a phrase like "the local" or "the local breed of" would fix a lot.

**Comment by esksp** [comment]
The confusion probably stems from naming "cow" something which is clearly some kind of monstrous centipede... :-D
We earthlings have a very precise picture of what a "cow" looks like, and a "longcow" would be just that, a long cow. Now one with segments, and additional legs? That doesn't fit. Cows don't have that.

Now if you change their name to "Wrghb" (or some such), all contradictions disappear instantly: Because of course Wrghbs are multi-segmented and can have _any_ amount of legs and _any_ character, who would question that?...

**Reply by Saugia** [reply]
I think this is a case where the more critical we get of it, the more we lose in terms of creativity. Remember, the Saryds are straight-up human centaurs. It's not an issue for some in the Coalition to refer to animals as cows, especially if this is through a translation device. I don't think we need to be this critical of something that is flavorful in lore. Many rather enjoy the silliness of "longcows".

You can almost think of it in cartoon-ish way, someone mentions "longcow" on some alien world, you expect the cartoon to show a cow-like alien, and it turns out to be something different entirely. ES is good at throwing out fun examples like this; the more we try and make sense of every little detail in the game for the player, the more I feel we squeeze the fun out of things like this.

**Reply by esksp** [reply]
> Remember, the Saryds are straight-up human centaurs.

Yes, that _is_ a little cheesy, especially since centaurs are, let say "biologically illogical"... But you forget that aspect quickly since it's mostly a text adventure, and "Saryds" are just another alien civilization living in a specific setting.

I know I _do_ eventually forget the Hai are giant squirrels and the Wanderers giant birds and the Remnants hearing-deficient people of African descent (and so on)... It's all extremely Earth-centric and not very credible, but since it's not an exobiology/ethnology game but a space adventure game, it isn't important. It's like movie aliens just being standard humans with strange skin colors and some tacked-on prosthetics, it doesn't prevent you from enjoying that movie.

> It's not an issue for some in the Coalition to refer to animals as cows, especially if this is through a translation device.

It remains, to put it mildly, "unlikely", especially coming from aliens who have no clue about human "cows", and thus wouldn't be tempted to give that name to their own, well-known animals (insects?).
Because the very concept of Earth animals must (should) be totally foreign to them, isn't it. I would had expected some local name, that would be the logical thing. Now _humans_ living there might _nickname_ them "longcows", but it definitely requires a human background to know what a "cow" is.
But as I said it's no big thing. It's just my analytical mind.

> you expect the cartoon to show a cow-like alien, and it turns out to be something different entirely.

OK, I won't spoil your fun, even if it is lost to me... :-)
I was just trying to explain that names have consequences, and in this case blatantly misnaming something might be a private joke to some, but it's to be expected that this joke will be lost to others. As this thread proves.
But again, it's _not_ a problem, just a minor head scratching issue without consequences.

**Reply by tuuktuuk** [reply]
I personally found it very funny when it turned out that the "cow" is a centipede. Of course I had already pictured a limousine cow in my head, and was delighted to find out that it's a very different looking animal :)

I found it a positive surprise, and I would not have gotten surprised if the animal's name had been wrghb.

**Reply by esksp** [reply]
> I personally found it very funny

I didn't, I just acknowledged the fact this alien giant centipede was called "Longcow" and went on. It's not like this is a very important issue.
As I said above, this is rather person-specific, not something most people would consider funny, so while some enjoy it, for others it falls flat. 🤷‍♂️ 

This is really just a rather unimportant setting detail (We could as well start debating over some system/planet name choices...), you could as well name those edible giant centipedes "Mary" for what I care... 🤷‍♂️

**Comment by Vlada3333** [comment]
Thank you for the information!
